frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/backgroundables/RebaseOnParentBackgroundable.java

```diff
@@ -99,41 +95,8 @@ public void doRun(ProgressIndicator indicator) {
       return;
     }
 
-    AtomicReference<Try<@Nullable ExecutionResult>> wrapper = new AtomicReference<>(
-        Try.success(null));
-
     val preRebaseHookExecutor = new PreRebaseHookExecutor(gitRepository);
-
-    new GitFreezingProcess(project, getTitle(), () -> {
```

Comment:
Whoops let's keep `GitFreezingProcess`... when someone e.g. triggers a commit (Cmd+K), this is what happens when an action is wrapped into `GitFreezingProcess`:

<img width="594" alt="image" src="https://user-images.githubusercontent.com/3383210/230719590-759ea209-88cb-455f-b380-1d271356fcb2.png">

Reply:
...which is desirable ofc

Reply:
wow, thanks for testing & verifying